### ISSUE TITLE
Determining optimal cut probabilities, etc. (classification)

### ISSUE THREAD

**Issue by shane-kercheval** [shane-kercheval/r-predictive-analysis-template#8]
Perhaps create evaluation set, used to determines ideal cutoff, (APM pg 425 "It is important, ..., to use an independent data set to derive the cutoff..."), although, i don't see a problem using training data since the final evaluation and measurement will still be on the test set. Perhaps the authors were referring to the assessments of sensitivity and specificity in the training set, which won't be an issue here since they aren't used.

- Use `Youden's J Index`, to determine cutoff
- Use 'costs' to find ideal cutoff (?)
